### DISCUSSION TITLE
Is there changelog for v1.14

### DISCUSSION THREAD

**Discussion by benjaminhuo** [kubeedge/kubeedge#4836] [General]
Cannot find changelog for v1.14
https://github.com/kubeedge/kubeedge/releases/tag/v1.14.0
https://github.com/kubeedge/kubeedge/tree/master/CHANGELOG

**Comment by ravaga** [comment]
You can find [here](https://github.com/kubeedge/kubeedge/blob/master/CHANGELOG/CHANGELOG-1.14.md#v1144) the changelog for v1.14.4
